src/main/java/io/getunleash/metric/DefaultHttpMetricsSender.java

```diff
@@ -33,6 +35,7 @@ public DefaultHttpMetricsSender(UnleashConfig unleashConfig) {
         this.gson =
                 new GsonBuilder()
                         .registerTypeAdapter(LocalDateTime.class, new DateTimeSerializer())
+                        .registerTypeAdapter(Instant.class, new InstantSerializer())
```

Comment:
Yggdrasil deals in Instants not Datetimes. This allows us to serialize those to ISO 8601 when we send metrics